lib/screens/authentication/import_key/import_words_screen.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:seeds/blocs/authentication/viewmodels/authentication_bloc.dart';
+import 'package:seeds/components/flat_button_long.dart';
+import 'package:seeds/design/app_theme.dart';
+import 'package:seeds/domain-shared/ui_constants.dart';
+import 'package:seeds/i18n/authentication/import_key/import_key.i18n.dart';
+import 'package:seeds/navigation/navigation_service.dart';
+import 'package:seeds/screens/authentication/import_key/interactor/import_key_bloc.dart';
+import 'package:seeds/screens/authentication/import_key/interactor/viewmodels/import_key_events.dart';
+import 'package:seeds/screens/authentication/import_key/interactor/viewmodels/import_key_state.dart';
+import 'package:seeds/utils/mnemonic_code/words_list.dart';
+
+const NUMBER_OF_WORDS = 12;
+const NUMBER_OF_COLUMNS = 3;
+
+class ImportWordsScreen extends StatefulWidget {
+  const ImportWordsScreen({Key? key}) : super(key: key);
+
+  @override
+  _ImportKeyScreenState createState() => _ImportKeyScreenState();
+}
+
+class _ImportKeyScreenState extends State<ImportWordsScreen> {
+  late ImportKeyBloc _importKeyBloc;
+
+  @override
+  void initState() {
+    super.initState();
+    _importKeyBloc = ImportKeyBloc(BlocProvider.of<AuthenticationBloc>(context));
+  }
```

Comment:
If this is this _importKeyBloc variable is your only var you don't need a statefulwidget at least you do not want to write BlocProvider.of<ImportKeyBloc>(context) over and over that is valid too i guess.

Reply:
Or do you also use some state mechanism (controlers, timers..) later?